examples/lessons/08/app.js

```diff
@@ -170,56 +175,68 @@ function addKeyboardHandler(canvas) {
   addEvents(canvas, {
     onKeyDown(e) {
       switch (e.key) {
-      case 'up':
-        xSpeed -= 0.02;
-        break;
-      case 'down':
-        xSpeed += 0.02;
-        break;
-      case 'left':
-        ySpeed -= 0.02;
-        break;
-      case 'right':
-        ySpeed += 0.02;
-        break;
-      // handle page up/down
-      default:
-        if (e.code === 33) {
-          cubePositionZ -= 0.05;
-        } else if (e.code === 34) {
-          cubePositionZ += 0.05;
-        }
+        case 'up':
```

Comment:
Nit: Are those "tabs"? Can we replace with "spaces"?

Reply:
I can work on the control component in next pull request